libbeat/docs/outputconfig.asciidoc

```diff
@@ -812,11 +813,13 @@ The cluster metadata contain the actual Kafka brokers events are published to.
 
 ===== `version`
 
-Kafka version ${beatname_lc} is assumed to run against. Defaults to 1.0.0.
+Kafka version {beatname_lc} is assumed to run against. Defaults to 1.0.0.
 
 Event timestamps will be added, if version 0.10.0.0+ is enabled.
 
-Valid values are all kafka releases in between `0.8.2.0` and `1.1.1`.
+Valid values are all kafka releases in between `0.8.2.0` and `2.0.0`.
+
+See <<kafka-compatibility,`Compatibility`>> for information on supported versions.
 
```

Comment:
I wouldn't hard code the link text. You only need the link text when you want to change the text that's used for the link. Use this instead:

```
<<kafka-compatibility>>
```

Reply:
I wouldn't worry about it. We can clean those up as we go. Mainly I just wanted you to know that the link text is not required if it duplicates the heading text.

Reply:
I'd assume then that this is ready to merge :) Waiting for jenkins...